src/status_im/chat/styles/input/emoji.cljs

```diff
@@ -0,0 +1,16 @@
+(ns status-im.chat.styles.input.emoji
+  (:require [status-im.components.styles :as common]))
```

Comment:
`:as` is definitely more explicit than `:refer`, so it's better. About using `:as` everywhere, this is more a matter of taste.